src/index.html

```diff
@@ -4,62 +4,62 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./3rd-party/Skeleton-2.0.4/css/normalize.css">
-  <link rel="stylesheet" href="./3rd-party/Skeleton-2.0.4/css/skeleton.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
   <title>Desk Buddy</title>
 </head>
+
 <body class="container">
-  <div class="seven columns">
+  <div class="main" id="top">
+
     <div class="header">
       <div class="buddy-pic">
         <img class="main-profile-pic" src="./assets/images/buddy-profile-pic-cat.png" alt="buddy profile pic">
       </div>
       <div class="profile-blurb">
-        <h5>Hey <span id="username">there</span>!</h5>
-        <p>I am Meowry - your new desk buddy! Don't worry, I don't bite 😸 (Although I may scratch).</p>
-        <p>Just poke me whenever you want a break~</p>
+        <h2>Hey <span id="username">there</span>!</h2>
```

Comment:
h1, h2 is used semantically. h1 means the most important text in the page. h2 means the 2nd most. 

Where is h1?